packages/calendar-utils/src/calendar-body/calendar-body.tsx

```diff
@@ -44,11 +44,15 @@ ClearSection.displayName = 'ClearSection';
 
 export type TInputProps = {
   /**
-   * a11y attribute to determine if input is valid/not
+   * Indicate if the value entered in the input is invalid.
+   * <br>
+   * [Props from React select was used](https://react-select.com/props)
```

Comment:
Sorry maybe I wasn't super clear in the previous comments. The link to react-select should only be added in case the prop is indeed passed to a react-select component.